addons/iap_mail/models/iap_account.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class IapAccount(models.Model):
+    _inherit = 'iap.account'
+
+    @api.model
+    def _send_iap_bus_notification(self, service_name, title, error_type=False):
+        param = {
+            'type': 'iap_notification',
```

Comment:
type in payload is no longer necessary, as it is already given as param to send method